settings.gradle.kts

```diff
@@ -45,3 +45,4 @@ rootProject.name = "spark-android"
 include(":spark")
 include(":sample")
 include(":spark-lint")
+include(":screenshots-test")
```

Comment:
nit: I'm not fond of this naming. I was thinking of using a common prefix for all spark modules. To me, `spark-screenshot-testing` would feel more natural. WDYT?

Reply:
Done in ca348372